project/src/main/world/creature/creature-visuals.gd

```diff
@@ -353,28 +352,6 @@ func play_movement_animation(animation_prefix: String, movement_direction: Vecto
 			$TailZ1.frame = 1 if orientation in [SOUTHWEST, SOUTHEAST] else 2
 
 
-"""
-Computes the nearest orientation for the specified direction.
-
-For example, a direction of (0.99, -0.13) is mostly pointing towards the x-axis, so it would result in an orientation
-of 'southeast'.
-"""
-func _compute_orientation(direction: Vector2) -> int:
-	if direction.length() == 0:
-		# we default to the current orientation if given a zero-length vector
-		return orientation
-	
-	# preserve the old orientation if it's close to the new orientation. this prevents us from flipping repeatedly
-	# when our direction puts us between two orientations.
-	var new_orientation: int = orientation
-	# unrounded orientation is a float in the range [-2.0, 2.0]
-	var unrounded_orientation := -2 * direction.angle_to(southeast_dir) / PI
-	if abs(unrounded_orientation - orientation) >= 0.6 and abs(unrounded_orientation + 4 - orientation) >= 0.6:
-		# convert the float orientation [-2.0, 2.0] to an int orientation [0, 3]
-		new_orientation = wrapi(int(round(unrounded_orientation)), 0, 4)
-	return new_orientation
```

Comment:
Reordering these methods was good; reorder the smooth path methods too.

Reply:
The smooth path methods appear to be in the correct order. It has some private/public intermingled fields, but there's an arguable purpose to that as it impacts the appearance of the editor.